modules.d/01fips/fips.sh

```diff
@@ -81,6 +94,33 @@ do_rhevh_check() {
     return 0
 }
 
+do_uki_check() {
+    KVER=$(uname -r)
```

Comment:
Shouldn't all local variables be declared with `local` to ensure there're no var collision?

Reply:
* this applies to all functions in this change